doc/modules/mod_inbox.md

```diff
@@ -5,19 +5,51 @@ To use it, enable mod\_inbox in the config file.
 
 ### Options
 
-* **backend** (atom, default: `rdbms`) - Database backend to use. For now, only `rdbms` is supported.
-* **reset_markers** (list, default: `[displayed]`) - List of atom chat markers that when sent, will reset the unread message counter for a conversation.
+#### `modules.mod_inbox.backend`
+* **Syntax:** string
+* **Default:** `rdbms`
+* **Example:** `backend = rdbms`
+
+Database backend to use. For now, only `rdbms` is supported.
+
+#### `modules.mod_inbox.reset_markers`
+* **Syntax:** array of strings
+* **Default:** `["displayed"]`
+* **Example:** `reset_markers = ["received"]`
+
+List of chat markers that when sent, will reset the unread message counter for a conversation.
 This works when [Chat Markers](https://xmpp.org/extensions/xep-0333.html) are enabled on the client side.
 Possible values are from the set: `displayed`, `received`, `acknowledged`. Setting as empty list (not recommended) means that no chat marker can decrease the counter value.
```

Comment:
We are a bit inconsistent here. Sometimes we say that `option = values: "val1", "val2", "val3"` in the syntax description and sometimes we do that as a note below. I think we should agree on one of these, what do you think?

Reply:
Yes we should and I'd prefer to have it enumerated in `Syntax` section. I think yesterday we agreed with @chrzaszcz on Slack that in this case I can remove these values from description, what prevents redundancy.